sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md

```diff
@@ -44,36 +43,41 @@ rename-rules:
   Etag: ETag|etag
   TCP: Tcp
   UDP: Udp
+  Noreuse: NoReuse
 
 override-operation-name:
   Location_ListCachedImages: GetCachedImagesWithLocation
   Location_ListCapabilities: GetCapabilitiesWithLocation
-  Location_ListUsage: GetUsageWithLocation
+  Location_ListUsage: GetUsagesWithLocation
+  Containers_ExecuteCommand: ExecuteContainerCommand
+  Containers_ListLogs: GetContainerLogs
+
+prepend-rp-prefix:
+  - Container
+  - Volume
+  - Capabilities
+  - CapabilitiesListResult
+  - Scheme
 
-directive:
-  - rename-model:
-      from: Container
-      to: ContainerInstanceContainer
-  - rename-model:
-      from: Volume
-      to: ContainerInstanceVolume
-  - rename-model:
-      from: Logs
-      to: ContainerLogs
-  - rename-model:
-      from: Event
-      to: ContainerEvent
-      
-  - from: containerInstance.json
-    where: $.definitions
-    transform: >
-      $.ContainerAttachResponse["x-ms-client-name"] = "ContainerAttachResult";
-      $.ContainerExecResponse["x-ms-client-name"] = "ContainerExecResult";
-      $.Capabilities["x-ms-client-name"] = "ContainerInstanceCapabilities";
-      $.ContainerGroupSubnetId.properties.id["x-ms-format"] = "arm-id";
-      $.InitContainerDefinition["x-ms-client-name"] = "InitContainerDefinitionContent";
-      $.LogAnalytics.properties.workspaceId["x-ms-format"] = "arm-id";
-  - from: swagger-document
-    where: $.definitions.ContainerHttpGet.scheme["x-ms-enum"]
-    transform: $["name"] = "ContainerInstanceScheme"
+rename-mapping:
+  Logs: ContainerLogs
+  Event: ContainerEvent
```

Comment:
Is `ConstainerEvent` still colliding with other RPs?

Reply:
No